### ISSUE TITLE
Nedostatky galerie v interní sekci oproti minulému dizajnu stránek

### ISSUE THREAD

**Issue by Grejdy** [bosancz/bosan.cz#402]
Narazil jsem na dvě věci, co postrádám v galerii:

- řazení: seznam alb je značně nepřehledný - připravovaná alba nejsou od zveřejněných nijak oddělena, vše je smíseno dohromady a jediný způsob hledání je jednoduchý vyhledávač, nebo prostě scrollovat a číst jeden název alba za druhým; největší problém je to na mobilu, kde se na obrazovku vejde jen sloupec s názvem akce, takže tam člověk nikdy nevidí současně datum, stav a název a celé to tím pádem působí jako jeden velký guláš). Myslím, že by se to dalo vylepšit několika způsoby: udělat nějaký filtr (třeba i podle zúčastněných oddílů, lidí, co se podíleli na tvorbě alba, roku, případně měsíce...), zavést něco, jako „poslední otevřené“ a „oblíbené“ (většině lidí stačí pár alb, na kterých se podílejí a zbytek je v interní sekci prostě nezajímá), upravit rozložení na mobilu do nějaké použitelnější podoby (třeba přidat možnost zvětšení a zmenšení stránky a dát minimálně název na víc řádků, aby se to na tu obrazovku vešlo), udělat možnost řazení (třeba jako je ve většině průzkumníků souborů - klikne se třeba na název sloupce„datum“ a vše se seřadí podle data sestupně, po druhém kliknutí vzestupně, po kliknutí na „název“ se seřadí podle abecedy od A do Z, při dalším opačně...).

- náhled alba: při změně vzhledu interní sekce zmizela možnost definovat, které tři fotky se budou zobrazovat ve veřejné sekci jako náhled pod názvem alba, nebo jsem tu možnost aspoň nenašel (v minulé verzi to byly  rozkliknutelné hvězdičky u každé fotky a vždy tři fotky měly tento symbol aktivní a bylo u něj pořadové číslo 1-3 - tyto tři fotky se pak zobrazovaly jako náhled v uvedeném pořadí). Bylo by super něco podobného znovu přidat (teď mi přijde, že si web prostě hodí kostkou  co tam dá). Hvězdičky si myslím, že fungovaly dobře. Kdyby se povedlo, aby člověk porozkliknutí hvězdičky rovnou vybral, kolikátá fotka to v náhledu bude a nemusel při výměně jedné fotky pět minut proklikávat hvězdičky, aby byly ve správném pořadí, tak by to bylo super. A ještě lepší ny bylo, kdyby byla větší vůle v tom, kolik fotek v tom náhledu bude (něco, jako 1-5 nebo 1-3) a když by člověk přidal, nebo ubral moc hvězdiček, tak by web místo odebrání (nebo přidání) jiné hvězdičky prostě zahlásil, že je hvězdiček moc/málo, a že je potřeba nejdřív ubrat/přidat jinou.

(snad jsem to napsal srozumitelně:-))

**Comment by SmallhillCZ** [comment]
Jo, to jsme zapomněli. Teď se vyberou imho první tři, co nahraješ.
